android/BOINC/app/src/main/res/values/strings.xml

```diff
@@ -366,6 +366,7 @@
     <string name="menu_run_mode_enable">Resume</string>
     <string name="menu_about">About</string>
     <string name="menu_help">Help</string>
+    <string name="menu_bug_report">Bug report</string>
```

Comment:
Please use 'Report Issue' to match the manager.

Reply:
Done.